vercel/turbo
# fix: update example lockfiles

### Description

Update various lockfiles so that users who copy examples directly don't face unexpected breakages due to outdated lockfiles.

Fixes #4316 (and I believe #4307)

### Testing Instructions

As part of this PR, I changed the example tests so we run them using the lockfiles that are checked in. This should help us catch breakages due to old versions. I believe it will become increasingly important if we roll out @tknickman's new version of `create-turbo` that allows users to specify an example to use as a template as the template includes these lockfiles.
